tensorboard/plugins/hparams/tf_hparams_sessions_pane/tf-hparams-sessions-pane.ts

```diff
@@ -169,4 +182,30 @@ class TfHparamsSessionsPane extends PolymerElement {
     type: Number,
   })
   _selectedTab: number = 0;
+  _tableTabClicked: () => void = () => {
+    this.logTabClick('Tab Clicked', 'Table');
+  };
+  _parrallelCoordsTabClicked: () => void = () => {
+    this.logTabClick('Tab Clicked', 'Parrellel Coords');
+  };
+  _scatterPlotMatrixTabClicked: () => void = () => {
+    this.logTabClick('Tab Clicked', 'Scatter Plot Matrix');
+  };
+
+  logTabClick: (action: string, tabName?: string) => void = (
+    action: string,
+    tabName?: string
+  ) => {
+    // If window['ga'] is defined, use it, otherwise any logging calls will be
+    // no-ops. window['ga'] is only defined in the hosted TensorBoard.
+    // @ts-ignore
+    this.ga = window['ga'] || function () {};
+    // @ts-ignore
+    this.ga('send', {
```

Comment:
Why set a property on `TfHparamsSessionsPane`? Wouldn't making this a const accomplish the same thing without any side effects?
```suggestion
    const analytics = window['ga'] || function () {};
    // @ts-ignore
    analytics('send', {
```

Reply:
I like it.